explainer.md

```diff
@@ -118,38 +129,40 @@ today, either as part of their onboarding experience, or as a permanently
 button to indicate interest, and see grant rates as high as 95% in the
 permission prompts that follow. For these developers, the permission element
-will be a drop-in replacement that is straightforward to adopt. Here are some
-real-life examples:
+will be a drop-in replacement that is straightforward to adopt and easy to
+polyfill on browsers which do not support the PEPC. Here are some real-life
+examples:
 
 ![](images/image4.png) \
-_Example 4: A video-conferencing site. Clicking on the "Enable camera" button triggers
-a camera permission request._
+*Example 4: A video-conferencing site. Clicking on the "Enable camera" button
+triggers a camera permission request.*
 
 ![](images/image5.png) \
-_Example 5: A search site. Clicking on "Use precise location" triggers a geolocation
-permission request._
+*Example 5: A search site. Clicking on "Use precise location" triggers a
+geolocation permission request.*
 
 ![](images/image6.png) \
-_Example 6: A messaging site, clicking on the "Enable Desktop Notifications" button
-triggers a push notifications permission request._
+*Example 6: A messaging site, clicking on the "Enable Desktop Notifications"
+button triggers a push notifications permission request.*
 
 We believe that enshrining such a user-initiated approach in standards can
 contribute to consistently better permission request flows across the web. This
 is because the permission element offers the following compelling advantages to
 users and developers alike:
 
-- It is **non-interruptive**: it is static, small, and contained in the content
-  area on the same z-level.
-- It is **discoverable**: it can be placed by the developer within the user's
-  focus of attention; with the locality making it easier to find and more
-  convenient to interact with.
-- It provides more **contextual information**: it has a visual manifestation as
-  opposed to being a procedural API, requiring developers to think about
-  integrating it into the user journey at UX design time, as opposed to being
-  left as an afterthought during implementation, resulting in knock-on effects
-  relating to clearer context.
-- It allows users to **revert** a previous "deny" decision if they have changed
-  their mind and are now interested in the feature that the site provides.
+-   It is **non-interruptive**: it is static, small, and contained in the
+    content area on the same z-level.
+-   It is **discoverable**: it can be placed by the developer within the user's
+    focus of attention; with the locality making it easier to find and more
+    convenient to interact with.
+-   It provides more **contextual information**: it has a visual manifestation
+    as opposed to being a procedural API, requiring developers to think about
+    integrating it into the user journey at UX design time, as opposed to being
+    left as an afterthought during implementation, resulting in knock-on effects
+    relating to clearer context.
```

Comment:
These are all achievable with `<button>` or any relevant HTML input (i.e., I would argue that, apart from maybe getting the free text label, there is limited advantage listed in these first 3 points over existing solutions). A developer would still need to do to support browsers that don't implement this feature:

```HTML
<permission type="geolocation"><button onclick="doGeo()">Use precise location</button></button>
```

Consider that image 5 has required significant styling for integration into the page's content:  
![Screenshot 2023-12-20 at 13 35 38](https://github.com/WICG/PEPC/assets/870154/e2d976de-7729-4039-af44-b50df964d385)

So it's a bit unconvincing that it would be an "afterthought during implementation, resulting in knock-on effects". A developer still needs to deal with the exact same situation in all cases.

Reply:
Extracted into https://github.com/WICG/PEPC/issues/8